billing/lib/stripe.js

```diff
@@ -0,0 +1,39 @@
+import * as DidMailto from '@web3-storage/did-mailto'
+
+/**
+ * @typedef {import('../lib/api.js').AccountID} AccountID
+ * @typedef {import('stripe').Stripe} Stripe
+ * @typedef {import('stripe').Stripe.CustomerSubscriptionCreatedEvent} CustomerSubscriptionCreatedEvent
+ */
+
+/**
+ *
+ * @param {Stripe} stripe
+ * @param {CustomerSubscriptionCreatedEvent} event
+ * @param {import('./api.js').CustomerStore} customerStore
+ * @returns {Promise<import('@ucanto/interface').Result<import('@ucanto/interface').Unit, import('@ucanto/interface').Failure>>}
+ */
+export async function handleCustomerSubscriptionCreated(stripe, event, customerStore) {
```

Comment:
The idea with `lib` was that we'd move it into w3up when billing settles. For _reporting_ usage to Stripe I explicitly kept that code out of lib, as the payment provider being used should be left up to the implementation.

Unless you feel strongly otherwise, can we please move this function back into `functions/stripe.js` in order to keep following that intention?

Reply:
ah makes sense - moved!